controller/src/beerocks/master/son_management.cpp

```diff
@@ -814,6 +814,14 @@ void son_management::handle_cli_message(Socket *sd,
         message_com::send_cmdu(sd, cmdu_tx);
         break;
     }
+    
+    //=======================
+    case beerocks_message::ACTION_CLI_OREN_PRINT: {
```

Comment:
stubs should be a separate preoperative commit or  at least mention it in the commit message .

Reply:
stubs removed